packages/systemd/systemd.spec

```diff
@@ -515,4 +748,3 @@ install -p -m 0644 %{S:4} %{buildroot}%{_cross_factorydir}%{_cross_sysconfdir}/i
 %exclude %{_cross_bindir}/systemd-resolve
 %exclude %{_cross_sbindir}/resolvconf
 
-%changelog
```

Comment:
Aren't this required any more? Should we clean the other specs? I remember there were some problems with RPM when the `changelog` wasn't included, probably that changed with RPM 4.20(?).

Reply:
I thought you were the one who told me `%changelog` was optional now. 😀

I don't think it's generated warnings in a while, since before RPM 4.20, though it did back when I was writing the initial set of specs.